### DISCUSSION TITLE
Missing build artifacts for 9.4.0

### DISCUSSION THREAD

**Discussion by BFergerson** [apache/skywalking#10580] [General]
It doesn't look like build artifacts were created for SkyWalking 9.4.0 (e.g. https://repo.maven.apache.org/maven2/org/apache/skywalking/server-core/). Is there a reason for this that I missed?

**Comment by wu-sheng** [comment]
My fault. Forgot to click the final release button on the nexus repository. 
Here you go. It would be 24-48 hours on maven central.
